components/woocommerce/woocommerce.app.mjs

```diff
@@ -198,15 +200,24 @@ export default {
     async listPaymentGateways() {
       return this.listResources("payment_gateways");
     },
-    async listProducts(page) {
-      return this.listResources(`products?page=${page}`);
+    async listProducts(params = null) {
+      const q = querystring.stringify(params);
```

Comment:
I guess `querystring` is deprecated. Would you mind using [`query-string`](https://www.npmjs.com/package/query-string) or something similar?